testdata/project-v3/config/default/kustomization.yaml

```diff
@@ -13,15 +13,16 @@ namePrefix: project-v3-
 #  someName: someValue
 
 bases:
+- ../manager
 - ../crd
 - ../rbac
-- ../manager
-# [WEBHOOK] To enable webhook, uncomment all the sections with [WEBHOOK] prefix including the one in 
+# +kubebuilder:scaffold:bases
```

Comment:
Could you please clarify what the marker here will solve? I mean, could you provide an example of this marker replaced?